client/state/data-layer/wpcom/sites/automated-transfer/initiate/index.js

```diff
@@ -0,0 +1,65 @@
+/**
+ * Internal dependencies
+ */
+import wpcom from 'lib/wp';
+import { initiateTransfer, requestEligibility } from 'state/automated-transfer/actions';
+import { transferStates } from 'state/automated-transfer/constants';
+import {
+	AUTOMATED_TRANSFER_INITIATE_REQUEST,
+} from 'state/action-types';
+
+/**
+ * Maps from API response to internal representation of automated transfer initialize request
+ *
+ * @param {Object} data API response data
+ * @returns {Object} Calypso eligibility information
+ */
+const fromApi = data => Object.assign(
+	{
+		id: data.transfer_id,
+	},
+	data.status === 'active' && { status: transferStates.START },
```

Comment:
this is a point of failed communication _I think_. the eligibility and initiate endpoints _appear_ to return a `status` value that ends up indicating the same _status_ of a transfer, yet they use different terms.

I have asked @retrofox and @vindl about this but we might need to get together all three of us and figure this out.

all things considered, this is why I prefer to write interface documents **before** coding so that we can be clear across teams and code-bases what to expect. code-first is like getting on the highway before identifying which onramp we need 😏